src/views/OrderDetail.vue

```diff
@@ -246,17 +247,18 @@ export default defineComponent({
       const uploadData = this.ordersList.items.filter((item: any) => {
         return item.isSelected;
       }).map((item: any) => {
-        return {
+        const orderItem = {
```

Comment:
We could directly return. WDYT?

Reply:
Yes sir, updated the code to return order item object directly.